### DISCUSSION TITLE
安装渲染器出现警告

### DISCUSSION THREAD

**Discussion by MinQo** [YunYouJun/hexo-theme-starry#246] [Q&A 疑难杂症]
安装渲染器出现警告
![image](https://user-images.githubusercontent.com/94370670/150447833-61b735c3-a92b-4418-91a5-d7bd6b1cacb4.png)

**Comment by YunYouJun** [comment]
这是渲染器的一些依赖过时了（正常现象），可以催促去插件仓库催促更新。
